api_app/services/airlock.py

```diff
@@ -401,6 +407,31 @@ async def delete_review_user_resource(
     return operation
 
 
+async def disable_user_resource(
```

Comment:
why not reuse the patch_user_resource func?

Reply:
what do you mean by reuse? `patch_user_resource` is used here

Reply:
I mean the function that is under the route, which is basically copied here , in this file /workspaces/AzureTRE/api_app/api/routes/workspaces.py

Reply:
changed, moved the shared code to one method and used in both places